### DISCUSSION TITLE
Prima installazione Home Assistant Core su Docker

### DISCUSSION THREAD

**Discussion by Hellcarrier** [indomus/forum#2369] [HUB: Home Assistant]
Ciao a tutti e vi ringrazio fin da subito per la pazienza :)

Sono alle prime armi con HA, ho seguito la guida per installazione su minipc e fino a momento di installare HO tutto è filato liscio.
Mi trovo al punto: Installare e configurare Home Assistant Core con Docker su Mini PC/Intel NUC/computer con Linux Debian
Ho fatto i seguenti passaggi :
sudo mkdir -p /home/deb/homeassistant
sudo apt-get update && sudo apt-get upgrade docker
I restanti comandi della verifico li evito perché sono passati.
E ora arriva il problema, al momento del comando :
docker run -d --name="home-assistant" -e "TZ=Europe/Rome" -v "/home/deb/homeassistant:/config" -v "/run/dbus:/run/dbus:ro" --net=host --restart always ghcr.io/home-assistant/home-assistant:stable
mi riporta l'errore del comando non riconosciuto e non riesco ad andare avanti.

Gentilmente mi potete dare qualche dritta plz ;)

**Comment by snakuzzo** [comment]
Sì ma se chiedi aiuto così ci vuole la sfera di cristallo.
Dai evidenza dell'errore...hai assegnato il gruppo docker all'utente `deb` ?
Hai provato a ricollegarti in SSH ?

Altra cosa...visto che vedo che sei proprio alle prime armi, se proprio vuoi usare docker, oltre a studiare come funziona docker, ti consiglio di usare docker-compose (o il pulgin compose)...così ti risulterà più semplice

**Comment by KC-inDomus** [comment]
> E ora arriva il problema, al momento del comando :
> docker run -d --name="home-assistant" -e "TZ=Europe/Rome" -v "/home/deb/homeassistant:/config" -v "/run/dbus:/run/dbus:ro" --net=host --restart always ghcr.io/home-assistant/home-assistant:stable
> mi riporta l'errore del comando non riconosciuto e non riesco ad andare avanti.

devi essere un po' più preciso...

**Comment by Hellcarrier** [comment]
Innanzitutto grazie mille, come vi ho detto sono alle prime armi, quindi ditemi quello che posso mostrarvi per capire dove sbaglio.
Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
Last login: Wed Mar  1 18:29:50 2023 from ............
deb@debian:~$ docker run -d --name="home-assistant" -e "TZ=Europe/Rome" -v "/home/deb/homeassistant:/config" -v "/run/dbus:/run/dbus:ro" --net=host --restart always ghcr.io/home-assistant/home-assistant:stable
-bash: docker: command not found



Avevo anche provato a installare direttamente hassios , andando contro a quello che era scritto nelle guide ma purtroppo una volta installato non mi riconosce la scheda wifi e non sono riuscito in nessun modo ad abilitarla.

**Reply by indomus** [reply]
non hai installato Docker, come nella guida è indicato di fare

**Reply by Hellcarrier** [reply]
Ho ricontrollato e per sicurezza ho rifatto la procedura della guida,

sudo apt-get update -y
Poi qui il primo errore:
deb@debian:~$ sudo apt-get upgrade -y
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 libseccomp-dev : Depends: libseccomp2 (= 2.5.4-1+b3) but 2.5.1-1+deb11u1 is installed
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
deb@debian:~$

Eseguo il comando che mi dice:

deb@debian:~$ apt --fix-broken install
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?
deb@debian:~$

Qualcosa che non va ;)

**Reply by snakuzzo** [reply]
Il comando che ti propone va lanciato con `sudo`

**Reply by Hellcarrier** [reply]
> Il comando che ti propone va lanciato con `sudo`

Riprovo, grazie mille :)

**Comment by Hellcarrier** [comment]
Vorrei ringraziare tutti per il supporto, sono arrivato alla fine, ora ho solo da imparare ancora :)

Grazie mille
